core/src/consensus/stake/mod.rs

```diff
@@ -193,7 +193,6 @@ mod tests {
 
         let mut state = metadata_for_election();
         increase_term_id_until(&mut state, 29);
-        state.add_balance(&address, 1000).unwrap();
```

Comment:
I think ignoring the test is better than executing the meaningless code. You can ignore the test function using the attribute "ignore". Let's remove these test when implementing stake modules.
```
#[test]
#[ignore]
```